src/components/DataDrawer/Chart/index.tsx

```diff
@@ -13,7 +13,7 @@ type ChartProps = {
 
 function colorShuffle(colors: string[]) {
   return colors.map((_, i) =>
-    (-1) ** i !== -1 ? colors[i] : colors[colors.length - i],
+    (-1) ** i !== -1 ? colors[i] : colors[colors.length - i - 1],
```

Comment:
is this `(-1) ** i !== -1` use to check the parity of `i`?

I commonly saw `i % 2` for that (return `0` for even and `1` for odd)

Reply:
yup that's better